build-system/tasks/runtime-test.js

```diff
@@ -60,6 +60,9 @@ function getConfig() {
   if (argv.ie) {
     return Object.assign({}, karmaDefault, {browsers: ['IE']});
   }
+  if (argv.chrome_canary) {
+    return Object.assign({}, karmaDefault, {browsers: ['ChromeCanary']});
```

Comment:
I assume you read https://github.com/karma-runner/karma-chrome-launcher#configuration, which says you can use chromium too. (Not sure if that's useful for AMP development.)

Reply:
Yup, I think we want to keep our reliance on Chromium minimum, since there are some (low risk albeit) mods from Chromium -> Chrome.